LondonDataServices.IDecide.Manage.Server/appsettings.json

```diff
@@ -105,7 +105,19 @@
         "PhoneNumber": "07700901890",
         "RegisteredGpPractice": "E19201"
       }
-    ]
+    ],
```

Comment:
See the previous comment on `TestWebApplicationFactory.cs` I do not think we should add config here that will polute out actual implementation deployed to `DEV` / `TEST` / `PRE-PROD` / `PROD`

I think we can remove these sections: `FakeFHIRProviderConfigurations` and `AcceptanceFakePatientRedactedPatient` and move them to the Acceptance Test project.

(If we intentionally need to use a fake provider in any deplyed environment, we can script this into the EnvironmentVariables section in Azure.  Or we can add this section, but then the content will be dictated by the testers)